src/common/dependency-manager/local-manager.ts

```diff
@@ -75,7 +75,7 @@ export default class LocalDependencyManager extends DependencyManager {
     });
 
     if (config.getDebugOptions()) {
-      node.command = config.getDebugOptions()?.command;
+      node.command = config.getDebugOptions() ?.command;
```

Comment:
There shouldn't be a space here.

Reply:
:grimacing: my IDE disagrees probably because it's a very new feature

Reply:
:grimacing: my IDE disagrees probably because it's a very new feature